binnacle/lib/repository.dart

```diff
@@ -159,4 +166,10 @@ class Repository {
   setActiveService(ServiceData serviceData) {
     _providerMap[serviceData.category].changeService(serviceData);
   }
+
+  void _setUpNavigationProvider(BehaviorSubject<PositionModel> positionStream) {
```

Comment:
Does this need its own function?

Reply:
I removed this thanks for pointing that out lol